modules/addons/elastic/manifests/es.yaml

```diff
@@ -40,7 +40,7 @@ spec:
   http:
     service:
       spec:
-        type: LoadBalancer
+        type: ClusterIP
```

Comment:
This is a small cleanup. I highly doubt anyone would be accessing Kibana directly to see what's there in Elastic/Skywalking. When that is done (quite uncommon) it's not a big deal to forward the port, and we save one Elastic IP.

Reply:
makes sense!